lib/modules/datasource/galaxy-collection/readme.md

```diff
@@ -0,0 +1,18 @@
+This datasource identifies ansible collection source repositories.
+By default [https://galaxy.ansible.com](https://galaxy.ansible.com) is used.
+
+Custom registries currently support the following:
+
+- setting your own host - this assumes the same API as `https://galaxy.ansible.com` uses, `/api/v2/collections` will be appended.
+- setting a complete custom URL including pathes - e.g. if you use a private automation hub. This URL is used as-is, so you have to provide the correct path where the collections can be found.
```

Comment:
```suggestion
- Setting your own host - this assumes the same API as `https://galaxy.ansible.com` uses, `/api/v2/collections` will be appended
- Setting a complete custom URL including paths, for example if you use a private automation hub. This URL is used as-is, so you have to provide the correct path where the collections can be found
```

- Start sentences with a capital letter
- Fix typo in word `pathes` -> `paths`
- We don't use `.` at the end of a list item in our docs
- I don't like `e.g.` and `i.e`, so let's use `for example` instead 😄